docker/run_tests.bash

```diff
@@ -8,4 +8,4 @@ docker run \
     --network="dbt-net" \
     -v $PWD/dbt:/root/.dbt \
     dbt-trino \
-    "cd /jaffle_shop && dbt seed && dbt run && dbt test"
+    "cd /jaffle_shop && dbt seed && dbt run && dbt run --model dim_order_dates && dbt test"
```

Comment:
Any idea here on how to split the command on multiple lines? 
I'd add a comment about the `dbt run --model dim_order_dates` which is used mainly to test the run with an `incremental` model.

Reply:
> Any idea here on how to split the command on multiple lines?

What do you mean here? You want to have separate test suites for incremental and everything else?

Reply:
No. I just mean to split the line content of what needs to be executed within docker on multiple lines for readability.

Reply:
got it, you can do something like:
```
docker run \
    --network="dbt-net" \
    -v $PWD/dbt:/root/.dbt \
    dbt-trino \
    "cd /jaffle_shop \
        && dbt seed \
        && dbt run \
        && dbt run --model dim_order_dates # some comment \
        && dbt test"

```